Bottom: 8dfb3bf67cb3f979aad56ab1a78c3a8033ac92e8
Top:    b590f61c655aed469991494ad9f9b2cd242f4e64
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-25 09:20:27 +0800

Refresh of autofs-5.0.7-fix-compile-error-with-heimdal-support-enabled.patch

---

diff --git a/modules/cyrus-sasl.c b/modules/cyrus-sasl.c
index 6115f90..b9c16f3 100644
--- a/modules/cyrus-sasl.c
+++ b/modules/cyrus-sasl.c
@@ -68,23 +68,27 @@
 void _krb5_princ_realm(krb5_context context, krb5_const_principal princ,
                           const char **realm, int *len)
 {
-    *realm = krb5_principal_get_realm(context, princ);
-    *len = strlen(*realm);
-}
+	*realm = krb5_principal_get_realm(context, princ);
+	if (*realm)
+		*len = strlen(*realm);
+	else
+		*len = 0;
+	return;
 #else
 void _krb5_princ_realm(krb5_context context, krb5_const_principal princ,
                           const char **realm, int *len)
 {
-    const krb5_data *data;
-
-    data = krb5_princ_realm(context, princ);
-    if (data) {
-        *realm = data->data;
-        *len = data->length;
-    } else {
-        *realm = NULL;
-        *len = 0;
-    }
+	const krb5_data *data;
+
+	data = krb5_princ_realm(context, princ);
+	if (data) {
+		*realm = data->data;
+		*len = data->length;
+	} else {
+		*realm = NULL;
+		*len = 0;
+	}
+	return;
 }
 #endif
